nebula.config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
   build: {},
   serve: {
     open: false,
+    flags: {
+      SENSECLIENT_IM_5036_VIZBUNDLE_STYLING: true,
```

Comment:
Is this flag for the org-chart styling only? If so, it would be better having ORG_CHART in the flag name. Maybe it is too late to change?

Reply:
Yes, the flag is only for org-chart. Initially it was meant to include the variance waterfall styling also, but the scope was changed afterwards and I didn't update the name. I will change it to IM_5036_ORGCHART_STYLING and update qlik-sense-desktop and qlik-sense-server.

Reply:
Hi, I've created a new PR https://github.com/qlik-oss/sn-org-chart/pull/354 where the feature flag has been changed and also following your suggestion about splitting I've only included the property panel and property migration.